### PR TITLE
Add support page, minor refactoring

### PR DESCRIPTION
**Pull request by tkyi**
- added Getting Started section, updated doc structure in README
- moved Setting up a Datastore to the Getting Started section
- added Support page to Community section

Resolves https://github.com/screwdriver-cd/guide/issues/14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,14 +51,15 @@ Simply add a new markdown document to the folder hierarchy in `docs`, and add an
 	* Datastores
 	* Schemas
 	* Authorization Model
-* Reference
+* Getting Started
+	* Setting Up a Datastore
```
Comment (stjohnjohnson) [inline README.md:55]: `up` or `Up`

File: docs/community/support.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Support
+
+## GitHub
+
```
Comment (stjohnjohnson) [inline docs/community/support.md]: Space or no space.

**Comment by stjohnjohnson** [comment]
👍

**Comment by Filbird** [comment]
<(👍 )>

**Pull request merged**